api/v1alpha1/ratelimit_types.go

```diff
@@ -29,14 +29,16 @@ type RateLimitSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// Foo is an example field of RateLimit. Edit RateLimit_types.go to remove/update
 	RateLimitConfig v1.ConfigMap `json:"ratelimitconfig"`
+	DomainConfig    string       `json:"domainconfig"`
```

Comment:
What would this be? `example.com` ?

Reply:
let me change that by

> NameDomainConfig

This will be the name of the configmap

I am thinking about this relationship

1:1

1 CR : 1 Configmap


is this ok for you ?

Reply:
That might work, but I do not have the full picture :sweat_smile: Let's try it we can always go back if we see it won't work for us